frontend/src/components/dashboard/create/create-modal/CreateModal.tsx

```diff
@@ -15,8 +15,13 @@ const CreateModal = () => {
   const history = useHistory()
   const [selectedChannel, setSelectedChannel] = useState(ChannelType.SMS)
   const [name, setName] = useState('')
+  const [isCreating, setIsCreating] = useState(false)
+
   async function handleCreateCampaign() {
     try {
+      if (isCreating) return
+      setIsCreating(true)
```

Comment:
Should we set this back to false in the `catch`?  In the case that `createCampaign` throws an error, user would not be able to try again.

Reply:
Oh yes missed that, thanks!